.github/ISSUE_TEMPLATE/decommissioning_v_1_0.md

```diff
@@ -0,0 +1,58 @@
+# Decommissioning a Linux Server  
+
+**Describe wich and why will be a server decommissioning**
+<p>ID-Service Now: <input type="text" size="20" maxlength="40" name="ID-Number" value="ID-Number" /></p>  <!-- put the ID Number here -->
```

Comment:
Do these html input tags actually work? Why use them at all? As all repos "should" have autoreferencing Service now tickets, I'd suggest to simply provide a hint regarding the syntax wherever a possible Service now ticket ID is helpful:

```suggestion

ID-Service Now: SNOW-IDxxx

```

Reply:
I tested this code with the markdown preview and it looked ok (see image). 
Unfortunately, how the template ultimately looks like (using this codes) can only be seen by applying the template.

If you agree, I would like to leave this code and test it. If it doesn't work then I'll apply your proposal.

![Bildschirmfoto 2024-09-13 um 12 09 55](https://github.com/user-attachments/assets/51398144-148d-474d-aba8-0f0ee2893357)

Reply:
ok, let‘s see what happens.